spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -13,6 +13,10 @@ help() {
     echo "    --password <password>        Password for the database user"
     echo "    --autogenpw                  Auto generate a password for the database user"
     echo "    --standalone                 Configure the database server independent of Uyuni or SUSE Manager Server Database"
+    echo "    --external-host              The database host when configuring the report database using an external database server"
+    echo "    --external-user              The user when configuring the report database using an external database server"
+    echo "    --external-password          The password when configuring the report database using an external database server"
+    echo "    --external-provider          The Cloud provider that is hosting the external report database"
```

Comment:
Do you expect differences between aws, google and azure ?

Reply:
well ATM, no since google/azure do not allow a superuser equivalent like AWS.  if that would ever change then yes i would expect the role name to not be rds_superuser so each provider would need special handling

Reply:
yes this should be left in as long as we are considering any other option besides AWS. It seems from comments from 
@admd @rjmateus and @moio  they want to leave it open. I will mark this as resolved for now.